Gemfile

```
@@ -6,7 +6,7 @@ gemspec
 group :test do
   gem 'activejob'
   gem 'aws-sdk-core', '~> 3'
-  gem 'aws-sdk-sqs'
+  gem 'aws-sdk-sqs', '1.65.0'
```

@mkeemon if you create an issue on the Moto side, could also link it here in the comment?

```suggestion
  # 1.65.0 works with XML + Moto
  # See https://github.com/ruby-shoryuken/shoryuken/pull/753#issuecomment-1822720647
  gem 'aws-sdk-sqs', '1.65.0'
```